### PR TITLE
2016 12 gem upgrades

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Was #1101

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -75,4 +75,7 @@ Style/FrozenStringLiteralComment:
 Rails/Output:
   Exclude:
     - 'config/unicorn.rb'
-    - 'db/seeds.rb'
\ No newline at end of file
+    - 'db/seeds.rb'
+
+Style/EmptyMethod:
```
Comment (Br3nda) [inline .rubocop.yml]: You can add ignores for the few files that have this, using:
`bundle exec rubocop --auto-gen-config`

Comment (CloCkWeRX) [inline .rubocop.yml]: Ah hah

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -14,8 +14,8 @@
 # users commonly want.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
-require "codeclimate-test-reporter"
```
Comment (pozorvlak) [inline spec/spec_helper.rb]: What's this change for?

**Comment by CloCkWeRX** [comment]
Check out
https://github.com/codeclimate/ruby-test-reporter/blob/master/CHANGELOG.md
(v1.0.0)




On 13 Dec 2016 5:16 AM, "pozorvlak" <notifications@github.com> wrote:

*@pozorvlak* commented on this pull request.
------------------------------

In spec/spec_helper.rb
<https://github.com/Growstuff/growstuff/pull/1126#pullrequestreview-12521171>
:

> @@ -14,8 +14,8 @@
 # users commonly want.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
-require <http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration-require>
"codeclimate-test-reporter"

What's this change for?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Growstuff/growstuff/pull/1126#pullrequestreview-12521171>,
or mute the thread

File: spec/spec_helper.rb (hunk 2 above)
Comment (CloCkWeRX) [inline spec/spec_helper.rb]: Check out
https://github.com/codeclimate/ruby-test-reporter/blob/master/CHANGELOG.md
(v1.0.0)

Comment (pozorvlak) [inline spec/spec_helper.rb]: Gotcha, thanks. I note that it says

> Now, you are responsible for running codeclimate-test-reporter as a separate step in your build.

do we need to update .travis.yml?

**Pull request merged**